packages/connectivity/connectivity_platform_interface/lib/src/enums.dart

```diff
@@ -7,7 +7,10 @@ enum ConnectivityResult {
   mobile,
 
   /// None: Device not connected to any network
-  none
+  none,
+
+  /// Unknown: The plugin wasn't able to determine the connectivity status of the device
+  unknown,
```

Comment:
You might want to update the `utils.dart` file. Otherwise it would return `none`.

Reply:
I made it return unknown on default. This follows the same pattern as the other util there.